tests/runtimes/grpc/__snapshots__/grpc_test.ts.snap

```diff
@@ -0,0 +1,295 @@
+export const snapshot = {};
+
+snapshot[`Typegraph using grpc 1`] = `
+'[
+  {
+    "types": [
+      {
+        "type": "object",
+        "title": "helloworld",
+        "runtime": 0,
+        "policies": [],
+        "config": {},
+        "as_id": false,
+        "properties": {
+          "greet": 1
+        },
+        "required": [
+          "greet"
+        ]
+      },
+      {
+        "type": "function",
+        "title": "func_7",
+        "runtime": 1,
+        "policies": [
+          0
+        ],
+        "config": {},
+        "as_id": false,
+        "input": 2,
+        "output": 5,
+        "materializer": 0,
+        "rate_weight": null,
+        "rate_calls": false
+      },
+      {
+        "type": "object",
+        "title": "object_2",
+        "runtime": 1,
+        "policies": [],
+        "config": {},
+        "as_id": false,
+        "properties": {
+          "name": 3
+        },
+        "required": []
+      },
+      {
+        "type": "optional",
+        "title": "optional_1",
+        "runtime": 1,
+        "policies": [],
+        "config": {},
+        "as_id": false,
+        "item": 4,
+        "default_value": null
+      },
+      {
+        "type": "string",
+        "title": "string_0",
+        "runtime": 1,
+        "policies": [],
+        "config": {},
+        "as_id": false
+      },
+      {
+        "type": "object",
+        "title": "object_5",
+        "runtime": 1,
+        "policies": [],
+        "config": {},
+        "as_id": false,
+        "properties": {
+          "message": 3
+        },
+        "required": []
+      }
+    ],
+    "materializers": [
+      {
+        "name": "grpc",
+        "runtime": 1,
+        "effect": {
+          "effect": "read",
+          "idempotent": true
+        },
+        "data": {
+          "method": "/helloworld.Greeter/SayHello"
+        }
+      },
+      {
+        "name": "predefined_function",
+        "runtime": 0,
+        "effect": {
+          "effect": "read",
+          "idempotent": true
+        },
+        "data": {
+          "name": "true"
+        }
+      }
+    ],
+    "runtimes": [
+      {
+        "name": "deno",
+        "data": {
+          "worker": "default",
+          "permissions": {}
+        }
+      },
+      {
+        "name": "grpc",
+        "data": {
+          "proto_file_content": "syntax = \\\\"proto3\\\\";\\\\n\\\\npackage helloworld;\\\\n\\\\nservice Greeter {\\\\n  rpc SayHello (HelloRequest) returns (HelloReply) {}\\\\n}\\\\n\\\\nmessage HelloRequest {\\\\n  string name = 1;\\\\n}\\\\n\\\\nmessage HelloReply {\\\\n  string message = 1;\\\\n}\\\\n",
```

Comment:
The proto file content could be large so, I suggest to upload it as an artifact instead of inlining. It is not required on this PR though, so just create a ticket for that.